### PR TITLE
Various things

### PR DESCRIPTION
**Pull request by nseam**
Snapshot of current module code.
- Some added fields (via Field Api) are unrelated to the current entites architecture and need to be modified/removed.
- I didn't ran CodeSniffer yet
- There may be some missing comments
- There are many entity hook function stubs, they may be needed later

### CODE REVIEW

File: template.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<h2>Welcome,</h2>
```
Comment (kenorb) [inline template.txt]: Is this file is used anywhere?

File: regression.module [hunk 2]
```diff
@@ -50,33 +59,614 @@ function regression_init() {
...
+  $template          = $settings['notifications_email_regression_template'];
+
+  $css_body          = 'padding: 12px;';
+  
+  $body              = '<div style="' . $css_body . '">' . regression_replacement_patterns_replace($new_revision, $template) . '</div>';
+  
+  // Newlines become a "display:block".
+  $body              = str_replace("\n", '<span style="display:block;"></span>', $body);
+  
+  file_put_contents('C:\body.htm', $body);
```
Comment (kenorb) [inline regression.module]: please remove the hardcoded path

File: includes/regression.admin.paths.edit.inc [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+    
+    if ($i < $num_segments - 1) {
+      // This is not the last segment, we're adding slash to separate segments.
+      $container['uniques'][] = array(
+        '#markup' => '<span style="font-size: 18px; margin: 0 5px;">/</span>',
+      );
+    }
+  }
+
+  if (0)
```
Comment (kenorb) [inline includes/regression.admin.paths.edit.inc]: if (0)?

File: regression.module (hunk 2 above)
Comment (nseam) [inline regression.module]: W dniu 2013-04-23 13:29, Rafal pisze:

> In regression.module:
> 
> > -  $revision_new_date = date('Y-m-d H:i:s', $new_revision->updated);
> > -  $revision_new_diff = regression_diff_revisions($old_revision, $new_revision);
> >   +
> > -  $settings          = regression_get_path_configuration($entry->path, TRUE);
> >   +
> > -  $recipients        = $settings['notifications_email_recipients'];
> > -  $template          = $settings['notifications_email_regression_template'];
> >   +
> > -  $css_body          = 'padding: 12px;';
> >   +
> > -  $body              = '<div style="' . $css_body . '">' . regression_replacement_patterns_replace($new_revision, $template) . '</div>';
> >   +
> > -  // Newlines become a "display:block".
> > -  $body              = str_replace("\n", '<span style="display:block;"></span>', $body);
> >   +
> > -  file_put_contents('C:\body.htm', $body);
> 
> please remove the hardcoded path
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/kenorb/regression/pull/1/files#r3913732.
> 
> Done

## 

/Kind regards / Pozdrawiam,
Adrian Kierzkowski/

File: libraries/finediff/finediff.php [hunk 4]
```diff
@@ -0,0 +1,687 @@
+<?php
```
Comment (kenorb) [inline libraries/finediff/finediff.php]: finediff needs to be installed separately into libraries dir, as the licence is not compatible and added the info in Reports that it's missing

File: regression.info [hunk 5]
```diff
@@ -1,5 +1,15 @@
...
 core = 7.x
-package = Development
-php = 5.3
+package = Regression
+version = 7.x-0.1
+project = regression
+dependencies[] = ctools
+dependencies[] = html2txt
+dependencies[] = mailsystem
+dependencies[] = mimemail
```
Comment (kenorb) [inline regression.info]: make this dependency optional (enhances[] = mimemail)

File: regression.info [hunk 6]
```diff
@@ -1,5 +1,15 @@
...
+description = Page regression testing
 core = 7.x
-package = Development
-php = 5.3
+package = Regression
+version = 7.x-0.1
+project = regression
+dependencies[] = ctools
+dependencies[] = html2txt
+dependencies[] = mailsystem
```
Comment (kenorb) [inline regression.info]: where this dependency is used?

**Comment by kenorb** [comment]
Please sort out the comments.

**Pull request merged**